app/models/user.rb

```diff
@@ -273,15 +273,16 @@ def do_soft_delete(organisation)
     end
     return save! if organisations.any? # only actually mark deleted when no orgas left
 
-    Anonymizer::Core.anonymize_record!(self)
-    receipts.each { |r| Anonymizer::Core.anonymize_record!(r) }
-    rdvs.each { |r| Anonymizer::Core.anonymize_record!(r) }
+    Anonymizer.anonymize_record!(self)
+    receipts.each { |r| Anonymizer.anonymize_record!(r) }
+    rdvs.each { |r| Anonymizer.anonymize_record!(r) }
```

Comment:
on pourrait améliorer les perfs de ces appels en faisant qqch comme `Anonymizer.anonymize_records "receipts", scope: Receipt.arel_table.where(Receipt.arel_table[:id].in(receipts.pluck(:id)))`